### DISCUSSION TITLE
How do I publish a package for apm to see?

### DISCUSSION THREAD

**Discussion by ciacob** [airsdk/apm#80] [Q&A]
Sorry, don't know where else to ask, and didn't find any obvious spot on the web where this question is answered.

> So, assuming that I have a general-use `SWC` library, and it is hosted here on GitHub, and I would like to publish it as a package that is discoverable and installable through `apm`, how do I proceed?

Thank you.

**Comment by marchbold** [comment]
Hi,

Best place to start is in the "creating packages" documentation:

https://github.com/airsdk/apm/wiki/Usage-CreatingPackages

You can find a template package here: https://github.com/airsdk/airpackage-swc-template

What is the library you are looking to publish? We are still curating content fairly strictly until we get a lot of these processes outlined completely.
